beesdoo_product/models/beesdoo_product.py

```diff
@@ -276,6 +283,20 @@ class BeesdooScaleCategory(models.Model):
     ]
 
 
+class BeesdooProductHazard(models.Model):
+    _name = "beesdoo.product.hazard"
+    _description = "beesdoo.product.hazard"
+
+    name = fields.Char()
+    type = fields.Selection(
+        [
+            ("fds", "FDS"),
+            ("hazard", "Specific hazard"),
+        ]
+    )
+    active = fields.Boolean(default=True)
```

Comment:
Why having active field ? I don't see the point for such case.

Reply:
I just copy the functionality from the BeesdooProductLabel model. But, yeah, it doesn't work well